pet-feeding/src/main/java/com/epam/servlets/PersonServlet.java

```diff
@@ -1,15 +1,41 @@
 package com.epam.servlets;
 
+import com.epam.dao.PersonDaoImpl;
+import com.epam.domain.Person;
+
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Enumeration;
+import java.util.List;
 
-public class PersonServlet extends HttpServlet{
+public class PersonServlet extends HttpServlet {
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        //String htmlResponse = "<html><h3>Welcome to Servlet!</h3></html>";
+        Enumeration<String> parameterList = req.getParameterNames();
+        if (parameterList.hasMoreElements()) {
+            String firstParameter = parameterList.nextElement();
+            if (firstParameter.equals("id")){
+                String[] parameterValueList = req.getParameterValues("id");
+                Integer firstParamValue = Integer. valueOf(parameterValueList[0]);
+                req.setAttribute("persons", PersonDaoImpl.instance().getById(firstParamValue));
```

Comment:
According to layered architecture, servlet should access service, not dao directly